hudi-spark/src/main/java/org/apache/hudi/OverwriteWithLatestAvroPayload.java

```diff
@@ -60,7 +60,18 @@ public OverwriteWithLatestAvroPayload preCombine(OverwriteWithLatestAvroPayload
   @Override
   public Option<IndexedRecord> combineAndGetUpdateValue(IndexedRecord currentValue, Schema schema) throws IOException {
     // combining strategy here trivially ignores currentValue on disk and writes this record
-    return getInsertValue(schema);
+    Object deleteMarker = null;
```

Comment:
@vinothchandar : do you think we should add this logic to HoodieRecordPayload interface instead of here?

Reply:
as a default method? do you mean, having a base method there which you can override as needed? 
I prefer to leave it to the payload implementation as is now.. the API supports ability to deletes anyway

Reply:
yes. OverwriteWithLatestAvroPayload is configurable right? What incase some user changes this to something else, then deletes in delta streamer may not work as expected.